src/pages/public_registration/PublicGeneralContent.tsx

```diff
@@ -91,6 +94,32 @@ export const PublicGeneralContent = ({ registration }: PublicRegistrationContent
   const scopusIdentifier = registration.additionalIdentifiers?.find((identifier) => identifier.sourceName === 'Scopus')
     ?.value;
 
+  const getContextPublisherIdentifierFromId = (id: string | null) => {
+    if (id !== null) {
+      const publisherId = getIdentifierFromId(id);
+      return publisherId;
+    }
+  };
+
+  const publisherIdentifier = getContextPublisherIdentifierFromId((publicationContext as ChapterPublicationContext).id);
```

Comment:
Husk at denne komponenten brukes for alle typer, så `as ChapterPublicationContext` og `as BookPublicationContext` kan føre til trøbbel for noen av de andre typene